system-contract-dapp-playground/src/components/contract-interaction/erc/erc-20/methods/TokenInformation.tsx

```diff
@@ -0,0 +1,69 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import OneLineMethod from '../common/OneLineMethod';
+import { ContractFactory, BaseContract } from 'ethers';
+
+interface PageProps {
+  contractFactory: ContractFactory<any[], BaseContract>;
+}
+
+const TokenInformation = ({ contractFactory }: PageProps) => {
+  return (
+    <div className="w-full">
+      {/* wrapper */}
+      <div className="flex flex-col gap-6">
+        {/* name */}
+        <OneLineMethod
+          title={'Token Name'}
+          outputValue={'Hedera'}
```

Comment:
Remove output value defaults

Reply:
I'm guessing instead they should be using set methods for useState hooks?

Reply:
Yes you're right. They will be updated to use dynamic values from on-chain smart contracts. Changes will be made in the next PR.

Reply:
Okay will wait on next PR